libs/sdk-core/src/models.rs

```diff
@@ -1164,6 +1179,14 @@ pub enum LnUrlCallbackStatus {
     },
 }
 
+/// Represents a LNURL-pay request.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct LnUrlPayRequest {
+    pub req_data: LnUrlPayRequestData,
+    pub amount_msat: u64,
+    pub comment: Option<String>,
+}
```

Comment:
We need documentation for the fields of LnUrlPayRequest

Reply:
Done